### PR TITLE
Add description for nbo.md

### PR DESCRIPTION
**Pull request by cmungall**
@pmidford @leechuck - look OK? Is everything else up to date?

Note you can also add a verbose description below the last `--`. You may want to also add something more verbose on the main nbo README.md

### CODE REVIEW

File: ontology/nbo.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ id: nbo
 contact:
   email: g.gkoutos@gmail.com
```
Comment (leechuck) [inline ontology/nbo.md]: This should probably be g.gkoutos@bham.ac.uk now.

**Comment by leechuck** [comment]
Contact email address should probably be g.gkoutos@bham.ac.uk or obo-behavior, or both. Otherwise, this looks great, thank you.

**Comment by mcourtot** [comment]
@leechuck: good to merge?

**Comment by leechuck** [comment]
Yes, please.

**Pull request merged**